### ISSUE TITLE
Address Type Check Error "Parameter 'el' implicitly has an 'any' type."

### ISSUE THREAD

**Issue by lucianmurmurache** [activist-org/activist#709]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

As mentioned in [PR706](https://github.com/activist-org/activist/pull/706) there seems to be a type check error, which is the following:
```bash
Error: components/sidebar/left/SidebarLeftFooter.vue(16,18): error TS7006: Parameter 'el' implicitly has an 'any' type.
Error:  Command failed with exit code 2: vue-tsc --noEmit
```

More info can be found [here](https://github.com/activist-org/activist/actions/runs/7916705317/job/21622449164)

**Comment by andrewtavis** [comment]
Thanks for making this, @lucianmurmurache! So helpful to be able to discuss this and have issues made. Thanks for sharing the load in such a substantial way 😊

**Comment by andrewtavis** [comment]
Closed via #710 🥳 Thank you, @lucianmurmurache! 🙌 I added a point to check the right sidebar functionality to the next dev sync, btw. We can discuss on the 24th and finalize #712 from there :)
